### PR TITLE
SSD VPS hosting

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

File: app/pages/hosting/ssd-vps.html [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- AWESOME VPS TOOLS -->
+<div class="container-fluid">
+  <div class="container s-pad-y-hard">
+    <!-- Title -->
+    <h1 class="title-header">AWESOME VPS TOOLS</h1>
```
Comment (AmirSavand) [inline app/pages/hosting/ssd-vps.html]: Not all caps.

File: app/pages/hosting/ssd-vps.html [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    <!-- Features -->
+    <div class="row icon-title">
+      <div class="col-lg-6 grid">
+        <div class="row">
+          <div class="col-sm-3">
+            <i class="fa fa-cogs" style="color: #16a085"></i>
+          </div>
+          <div class="col-sm-9">
+            <h3>Instance And Free Setup</h3>
+            <p>
```
Comment (AmirSavand) [inline app/pages/hosting/ssd-vps.html:81]: One line.

File: app/pages/hosting/ssd-vps.html [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+            <h3>When Should I Move To A Virtual Server?</h3>
+            <p>
+              If you’re outgrowing your web hosting or simply want more control, a managed or fully managed VPS (Virtual
+              Private Server) is the next logical choice for your hosting needs.
+            </p>
+            <p>
+              It gives you more freedom with SSH Access to Apache and PHP, plus you can install an SSL certificate, as
+              well as any type of software – all without the responsibility of a physical dedicated server.
+            </p>
+            <p>
```
Comment (AmirSavand) [inline app/pages/hosting/ssd-vps.html:246]: One Line

**Review by AmirSavand — CHANGES_REQUESTED** [review]

File: app/pages/hosting/ssd-vps.html (hunk 2 above)
Comment (ArsalanSavand) [reply app/pages/hosting/ssd-vps.html:81]: 121

File: app/pages/hosting/ssd-vps.html (hunk 3 above)
Comment (ArsalanSavand) [reply app/pages/hosting/ssd-vps.html:246]: 121

File: app/pages/hosting/ssd-vps.html (hunk 1 above)
Comment (ArsalanSavand) [reply app/pages/hosting/ssd-vps.html]: Done.

**Review by AmirSavand — APPROVED** [review]

**Pull request merged**